kitchen.yml

```diff
@@ -48,6 +40,8 @@ suites:
       ganeti:
         master-node: true
         drbd: true
+    provisioner:
+      multiple_converge: 3 # to fix directory ownership contention
```

Comment:
Looks like this changed with the release of Ganeti 3.x and probably should be fixed in the recipe.
```
         * directory[/var/lib/ganeti/rapi] action create
           - change owner from 'gnt-rapi' to 'root'
           - change group from 'gnt-masterd' to 'root'
           - restore selinux security context
```